hack/verify-spelling.sh

```diff
@@ -24,7 +24,7 @@ CONTAINERIZE_IMAGE=quay.io/kubermatic/build:go-1.20-node-18-8 containerize ./hac
 echodate "Running codespell..."
 
 codespell \
-  --skip .git,_build,_dist,vendor,go.mod,go.sum,*.jpg,*.jpeg,*.png,*.woff,*.woff2,*.pem,node_modules,src,.angular,package-lock.json,dist,coverage,swagger.json,src \
+  --skip .git,_build,_dist,vendor,go.mod,go.sum,*.jpg,*.jpeg,*.png,*.woff,*.woff2,*.pem,node_modules,src,.angular,package-lock.json,dist,coverage,swagger.json,src,./modules/api/pkg/test/e2e/utils/apiclient \
```

Comment:
Why do we need this change?

Reply:
ah, then I guess there's not much we can do.

Reply:
@ahmadhamzh why did we omit the whole directory if the issue is just in a single file?

Reply:
@ahmedwaleedmalik Ahmad was following my suggestion - My idea was that the whole `modules/api/pkg/test/e2e/utils/apiclient` is generated, i.e. any typos we _can_ control would show up in other places of the code, and any typos in this directory and its subdirectories would stem from `go-swagger`.

Reply:
Yeah, but that wasn't really required since it was just one file that had a typo. Anyways it was already merged in the main branch so I couldn't care less.